.github/workflows/test.yml

```diff
@@ -0,0 +1,33 @@
+name: Tests
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+jobs:
+  rails:
+    strategy:
+      fail-fast: false
+      matrix:
+        rubygems_version: ['3.1.5', 'latest']
+        ruby_version: ['2.7.2']
+    name: Rails tests (Ruby ${{ matrix.ruby_version }}, RubyGems ${{ matrix.rubygems_version }})
+    runs-on: ubuntu-18.04
+    env:
+      RUBYGEMS_VERSION: ${{ matrix.rubygems_version }}
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install and start services
+      run: |
+        ./script/install_toxiproxy.sh
+        docker-compose up -d
```

Comment:
Notice this is done as early as possible to get some time for Elasticsearch to start.

Reply:
On travis we had to setup chrome explicitly to make js tests work. I am not sure how is github actions working.. perhaps we should add https://github.com/marketplace/actions/setup-chromedriver anyway?

Reply:
Github actiions provides this software by default: https://github.com/actions/virtual-environments/blob/4b03527e45271076bec1d4fdd4b4cfaaf3d3c4e7/images/linux/Ubuntu1804-README.md#browsers-and-drivers.

Reply:
nice. thank you for checking.